Smarty/templates/BrowserVariables.tpl

```diff
@@ -12,7 +12,7 @@
 var gVTModule = "{$GVTMODULE}";
 var gVTModuleLabel = "{$GVTMODULELABEL}";
 var gVTTheme  = "{$THEME}";
-var gVTModuleLabel = "{$GVTMODULELABEL}";
+var gVTModuleLabel = "{$GVTMODULELABEL}"; 
```

Comment:
This commit is incorrect. The extra space violates our guidelines. My comment was "next time". Next time separate the commit as it is a semantically separate change from the menu. Now it is to late to change the commit (or o costly to do so), so I was just trying to call your attention to being clean with your commits and changes, it will help you and us all going forward. Please revert this commit.